### PR TITLE
fix(apple): Clarify source context uploads code

### PR DESCRIPTION
**Pull request by philipphofmann**
Specify clearly that include sources uploads users' source code to Sentry.

### CODE REVIEW

File: src/platform-includes/debug-symbols-apple/_default.mdx [hunk 1]
```diff
@@ -34,7 +35,8 @@ Visit the [sentry-cli docs](/product/cli/dif/#uploading-files) for more informat
 
 If you're already using Fastlane, you can use the [Sentry Fastlane plugin](https://github.com/getsentry/sentry-fastlane-plugin)
 to upload your dSYMs to Sentry.
-For <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink> to work, we recommend setting the `include_sources` parameter to `true`:
+
+Sentry can display snippets of your code next to the events stacktraces. This feature is called <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink>. When setting the setting the `include_sources` parameter to `true`, the Fastlane plugin will scan your debug files to find references to the source code files, resolve them in the local file system, bundle them up, and upload them to Sentry.
```
Comment (lizokm) [inline src/platform-includes/debug-symbols-apple/_default.mdx:39]:
```suggestion
Sentry can display snippets of your code next to event stacktraces. This feature is called <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink>. When setting the setting the `include_sources` parameter to `true`, the Fastlane plugin will scan your debug files to find references to the source code files, resolve them in the local file system, bundle them up, and upload them to Sentry.
```

Comment (lizokm) [inline src/platform-includes/debug-symbols-apple/_default.mdx:39]:
```suggestion
Sentry can display snippets of your code next to event stacktraces. This feature is called <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink>. When setting the `include_sources` parameter to `true`, the Fastlane plugin will scan your debug files to find references to the source code files, resolve them in the local file system, bundle them up, and upload them to Sentry.
```

File: src/platform-includes/debug-symbols-apple/_default.mdx [hunk 2]
```diff
@@ -19,7 +19,8 @@ If this is your situation, please read [Info.plist Is Missing in Xcode 13 — He
 ## Sentry-cli
 
 Use the `sentry-cli upload-dif` command to upload dSYMs to Sentry. The command will recursively scan the provided folders or ZIP archives. Files that have already been uploaded will be skipped automatically.
-For <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink> to work, we recommend setting the `--include_sources` option.
+
+Sentry can display snippets of your code next to the events stacktraces. This feature is called <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink>. When setting the `--include_sources` option, sentry-cli will scan your debug files to find references to the source code files, resolve them in the local file system, bundle them up, and upload them to Sentry.
```
Comment (lizokm) [inline src/platform-includes/debug-symbols-apple/_default.mdx:23]:
```suggestion

Sentry can display snippets of your code next to event stacktraces. This feature is called <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink>. When setting the `--include_sources` option, sentry-cli will scan your debug files to find references to the source code files, resolve them in the local file system, bundle them up, and upload them to Sentry.
```

File: src/platform-includes/debug-symbols-apple/_default.mdx [hunk 3]
```diff
@@ -64,7 +66,7 @@ Follow these steps to upload the debug symbols:
 1.  Download and install [sentry-cli](/product/cli/installation/).
 2.  Copy the script below into a new _Run Script_ and set your _AUTH_TOKEN_, _ORG_SLUG_, and _PROJECT_SLUG_.
 
-For <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink> to work, we recommend setting the `--include_sources` option. Simply change `sentry-cli upload-dif` to `sentry-cli upload-dif --include_sources` in the copied script.
+Sentry can display snippets of your code next to the events stacktraces. This feature is called <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink>. When setting the `--include_sources` option, sentry-cli will scan your debug files to find references to the source code files, resolve them in the local file system, bundle them up, and upload them to Sentry. Simply change `sentry-cli upload-dif` to `sentry-cli upload-dif --include_sources` in the copied script.
```
Comment (lizokm) [inline src/platform-includes/debug-symbols-apple/_default.mdx:69]:
```suggestion
Sentry can display snippets of your code next to event stacktraces. This feature is called <PlatformLink to="/data-management/debug-files/source-context/">source context</PlatformLink>. When setting the `--include_sources` option, sentry-cli will scan your debug files to find references to the source code files, resolve them in the local file system, bundle them up, and upload them to Sentry. Simply change `sentry-cli upload-dif` to `sentry-cli upload-dif --include_sources` in the copied script.
```

**Review by lizokm — APPROVED** [review]
LGTM

File: src/platform-includes/debug-symbols-apple/_default.mdx (hunk 1 above)
Comment (philipphofmann) [reply src/platform-includes/debug-symbols-apple/_default.mdx]: Ups, sorry about that obvious typo.

**Comment by philipphofmann** [comment]
Thank you, @lizokm, for the review 😀.

**Pull request merged**